### DISCUSSION TITLE
config里面的todo是？

### DISCUSSION THREAD

**Discussion by WakiJi** [Womsxd/MihoyoBBSTools#201] [Q&A] [answered]
https://github.com/Womsxd/MihoyoBBSTools/blob/02f9a1813d6f332c82193539a6cabe599b9fd3de/config/config.yaml.example#L5-L6

**Comment by Womsxd** [comment, marked as answer]
给单个配置文件指定一个推送的配置文件(

**Reply by WakiJi** [reply]
用法是什么，直接在里面填入push.ini的路径吗（
